edb/lib/schema.edgeql

```diff
@@ -167,11 +167,11 @@ CREATE ABSTRACT TYPE schema::CallableObject
 
 
 CREATE ABSTRACT TYPE schema::VolatilitySubject EXTENDING schema::Object {
-    CREATE PROPERTY volatility -> schema::Volatility {
-        # NOTE: this default indicates the default value in the python
-        # implementation, but is not itself a source of truth
-        SET default := 'Volatile';
-    };
+    CREATE PROPERTY volatility -> schema::Volatility;
+    CREATE PROPERTY inferred_volatility -> schema::Volatility;
+    CREATE REQUIRED PROPERTY effective_volatility := (
```

Comment:
I don't think this is needed given that you already have the `get_effective_volatility()` helper function.

Reply:
We don't populate these tables with that sort of helper function value, though

Reply:
That was my point -- let's not add properties to `schema::` unless we need to. And it seems like `effective_volatility` can be computed in Python whenever we need it, so there's no strong need to have it in `schema::`.

Reply:
It's not really required for anything to work, but the idea for adding it was so that the correct value is available via reflection easily